packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -1,37 +1,161 @@
+import contextlib
+import json
 import logging
+import random
+import threading
+from collections import deque
 from subprocess import check_output
 
 import pytest
-
 import requests
-
 import retrying
 
 from pkgpanda.build import load_json
 from test_util.marathon import get_test_app, get_test_app_in_docker
 
 
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
```

Comment:
Don't leave this log.setLevel(logging.INFO) in this commit. We want this logger to inherit the root logging level (which is already at INFO)